### ISSUE TITLE
WMS fails in ArcGIS 10.3

### ISSUE THREAD

**Issue by landam** [ctu-osgeorel/subdayprecip-design#4]
When adding WMS to existing project, ArcGIS fails with

```
One or more layers failed to draw: Rain WMS server: WMS service exceptions:-Service Error Code:InvalidSRS Service Error Description: msWMSLoadGetMapParams(): WMS server error. Invalid CRS given : CRS must be valid for all requested layers. msProcessProjection(): Projection library error. proj error -Service Error Code:InvalidSRS Service Error Description: " -Service Error Code:InvalidSRS Service Error Description: no options found in -Service Error Code:InvalidSRS Service Error Description: ' -Service Error Code:InvalidSRS Service Error Description: init -Service Error Code:InvalidSRS Service Error Description: ' -Service Error Code:InvalidSRS Service Error Description: file -Service Error Code:InvalidSRS Service Error Description: " -Service Error Code:InvalidSRS Service Error Description: for -Service Error Code:InvalidSRS Service Error Description: " -Service Error Code:InvalidSRS Service Error Description: init=epsg:102067 -Service Error Code:InvalidSRS Service Error Description: " -Service Error Code:InvalidSRS -Service Error Code:InvalidSRS
```

**Comment by landam** [comment]
see discussion at https://lists.osgeo.org/pipermail/mapserver-users/2015-November/078428.html

**Comment by landam** [comment]
Solved in 802c5548bc4f1fc54d
